src/test/java/io/lettuce/core/json/RedisJsonIntegrationTests.java

```diff
@@ -163,8 +163,16 @@ public void jsonArrpopEmptyArray() {
         redis.jsonSet("myKey", JsonPath.ROOT_PATH, value);
         List<JsonValue> result = redis.jsonArrpop("myKey");
         assertThat(result.toString()).isEqualTo("[\"one\"]");
-        result = redis.jsonArrpop("myKey", JsonPath.ROOT_PATH, 0);
-        assertThat(result.get(0).isNull()).isTrue();
+        assertThat(redis.jsonGet("myKey").toString()).isEqualTo("[[]]");
+    }
+
+    @Test
+    public void jsonArrpopWithRootPathAndIndex() {
+        JsonValue value = redis.getJsonParser().createJsonValue("[\"one\", \"two\", \"three\"]");
+        redis.jsonSet("myKey2", JsonPath.ROOT_PATH, value);
+        List<JsonValue> result = redis.jsonArrpop("myKey2", JsonPath.ROOT_PATH, 1);
+        assertThat(result.toString()).isEqualTo("[\"two\"]");
```

Comment:
@tishun I try to add the test, but in fails in this with the message 
```
RedisJsonIntegrationTests.jsonArrpopWithRootPathAndIndex:173 » RedisCommandExecution ERR Path '$.1' does not exist or not an array
```
Could you explain me the reason?
By the way, I can not run the Integration Test on local by run `make test`.